src/atoms/Text/Text.web.js

```diff
@@ -1,3 +1,109 @@
-const Text = () => null;
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+import baseTheme from '../../tokens/theme';
+import { getColorKeys, getColor, getLineHeight } from '../../_helpers/theme';
+
+const styles = {
+  fontFamily({ theme, _weight }) {
+    return theme.fonts.family.lato[_weight];
+  },
+  fontSize({ theme, size }) {
+    return theme.fonts.size[size];
+  },
+  color({ theme, color }) {
+    return getColor(theme, color);
+  },
+  align({ align }) {
+    switch (align) {
+      case 'left':
+      default:
+        return 'flex-start';
+      case 'center':
+        return 'center';
+      case 'right':
+        return 'flex-end';
+    }
+  },
+  letterSpacing({ theme, _letterSpacing }) {
+    return `${theme.fonts.letterSpacing[_letterSpacing]}`;
+  },
+  lineHeight({ theme, size, _lineHeight }) {
+    if (_lineHeight) {
+      return theme.fonts.lineHeight[_lineHeight];
+    } else {
+      return getLineHeight(theme, size);
+    }
+  },
+  maxHeight({ theme, truncate, _lineHeight, maxLines }) {
+    if (truncate) {
```

Comment:
Let's remove the ```truncate``` prop and do this instead
```suggestion
    if (isDefined(maxLines)) {
```

Reply:
maxLines is always defined, since we have it in the default values, lmk if i am wrong

Reply:
done